python/lsst/ts/m2gui/model.py

```diff
@@ -52,6 +53,8 @@ class Model(object):
         Telemetry port to connect. (the default is 50011)
     timeout_connection : `int` or `float`, optional
         Connection timeout in second. (the default is 10)
+    is_simulation_mode: `bool`, optional
+        Is the simulation mode or not. (the default is False)
```

Comment:
True if running in simulation mode. Defaults to False.

Reply:
Updated in the commit: [7d7ab7f](https://github.com/lsst-ts/ts_m2gui/pull/17/commits/7d7ab7f36a1d11880485b59c021ba2289c02f003). Thanks!